biocypher_metta/adapters/gencode_exon_adapter.py

```diff
@@ -18,13 +18,13 @@ class GencodeExonAdapter(Adapter):
     ALLOWED_KEYS = ['gene_id', 'transcript_id', 'transcript_type', 'transcript_name', 'exon_number', 'exon_id']
     INDEX = {'chr': 0, 'type': 2, 'coord_start': 3, 'coord_end': 4, 'info': 8}
 
-    def __init__(self, write_properties, add_provenance, filepath=None,
+    def __init__(self, write_properties, label, add_provenance, filepath=None,
                  chr=None, start=None, end=None):
         self.filepath = filepath
         self.chr = chr
         self.start = start
         self.end = end
-        self.label = 'exon'
+        self.label = label if label else 'exon'
```

Comment:
instead you can put `label='exon'` in the constructor argument. But you should switch the order with `add_provenance`.
`def __init__(....,add_provenance, label='exon', ...)`